core/services/job/models.go

```diff
@@ -214,9 +214,10 @@ type OffchainReportingOracleSpec struct {
 	ContractConfigTrackerPollIntervalEnv      bool
 	ContractConfigConfirmations               uint16 `toml:"contractConfigConfirmations"`
 	ContractConfigConfirmationsEnv            bool
-	EVMChainID                                *utils.Big `toml:"evmChainID" db:"evm_chain_id"`
-	CreatedAt                                 time.Time  `toml:"-"`
-	UpdatedAt                                 time.Time  `toml:"-"`
+	EVMChainID                                *utils.Big      `toml:"evmChainID" db:"evm_chain_id"`
+	OCRDatabaseTimeout                        models.Interval `toml:"ocrDatabaseTimeout"`
```

Comment:
Could this just be called `databaseTimeout` since it is already within the context of an OCR spec?

Reply:
Fixed.